Cargo.toml

```diff
@@ -24,25 +27,49 @@ repository = "https://github.com/graphprotocol/graph-node"
 license = "MIT OR Apache-2.0"
 
 [workspace.dependencies]
-diesel = { version = "2.1.3", features = ["postgres", "serde_json", "numeric", "r2d2", "chrono"] }
+anyhow = "1.0"
+async-graphql = { version = "7.0.6", features = ["chrono", "uuid"] }
+async-graphql-axum = "7.0.6"
+axum = "0.7.5"
+chrono = "0.4.38"
+clap = { version = "4.5.4", features = ["derive", "env"] }
+constant_time_eq = "0.3.0"
+diesel = { version = "2.1.3", features = ["postgres", "serde_json", "numeric", "r2d2", "chrono", "uuid"] }
 diesel-derive-enum = { version = "2.1.0", features = ["postgres"] }
-diesel_derives = "2.1.4"
 diesel-dynamic-schema = "0.2.1"
+diesel_derives = "2.1.4"
 diesel_migrations = "2.1.0"
+graph = { path = "./graph" }
+graph-core = { path = "./core" }
+graph-store-postgres = { path = "./store/postgres" }
+graphman = { path = "./core/graphman" }
+graphman-extensions = { path = "./core/graphman_extensions" }
+graphman-primitives = { path = "./core/graphman_primitives" }
```

Comment:
After the refactor there are still 2 crates, one specific to graphman commands and one to avoid cyclic dependencies between graphman commands and postgres store.